frontend/src/organizations/components/OrganizationForm.tsx

```diff
@@ -0,0 +1,91 @@
+import { AxiosError, AxiosResponse } from 'axios';
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  Button,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Spinner,
+  VStack,
+} from '@chakra-ui/react';
+import { useFormik } from 'formik';
+import { useState } from 'react';
+import { sseHandler } from '@notifications/helpers/sse-handler';
+import { useSseSubscription } from '@notifications/hooks/api';
+import { ApplicationErrorDto } from '@shared/dtos/application-error';
+import { applicationErrorHandler } from '@shared/helpers/application-error';
+import { mapValidationErrors } from '@shared/mappers/application-error';
+import { CreateOrganizationDto, OrganizationCreatedDto } from '@organizations/dtos';
+import { useCreateOrganization } from '@organizations/hooks/api';
+
+const initialValues: CreateOrganizationDto = {
+  name: '',
+};
+
+export const OrganizationForm: React.FC = () => {
+  const { mutate: createOrganization, isSuccess, isLoading } = useCreateOrganization();
+  const [isOrganizationCreatedEventReceived, setIsOrganizationCreatedEventReceived] =
+    useState(false);
+  const [handler, setHandler] = useState(sseHandler());
+  useSseSubscription(handler);
+
+  const handleSubmitForm = (dto: CreateOrganizationDto): void =>
+    createOrganization(
+      { dto },
+      { onError: handleCreateOrganizationFailure, onSuccess: handleCreateOrganizationSuccess }
+    );
+
+  const { errors, touched, values, handleSubmit, handleChange, setErrors, resetForm } =
+    useFormik<CreateOrganizationDto>({
+      initialValues,
+      onSubmit: handleSubmitForm,
+    });
+
+  const handleCreateOrganizationFailure = (
+    error: AxiosError<ApplicationErrorDto<any, any>, unknown>
+  ): void =>
+    applicationErrorHandler<CreateOrganizationDto>()
+      .onGenericValidationFailed(({ details }) => setErrors(mapValidationErrors(details)))
+      .handleAxiosError(error);
+
+  const handleCreateOrganizationSuccess = (
+    response: AxiosResponse<OrganizationCreatedDto, CreateOrganizationDto>
+  ): void => {
+    setIsOrganizationCreatedEventReceived(false);
+    setHandler(
+      handler.onOrganizationCreatedEvent(({ data }) => {
+        handleOrganizationCreatedEvent();
```

Comment:
To powinno sie wykonywac tylko wtedy, jak przyjde sse z organizacja o danym id. Teraz sie wykonuje dla wszystkich sse